File: lib/sort.rb
class Array

  # my_array = [78, 23, 69, 10, 23]

  def bubble_sort
      count = 0
      while count < self.length-1 do
        i = 0
        while i < self.length-1 do
          if self[i] > self[i+1]
              self[i], self[i+1] = self[i+1], self[i]
          end
          i+=1
        end
        count += 1
      end
      return self
  end

  # bubble_sort(my_array)

end

File: sort.js
function bubbleSort(arr){
  var i,
      count;
  for (count = 0; count < arr.length-1; count++) {
      console.log(count);
      i=0;
    for (i = 0; i < arr.length-1; i++) {
      if (arr[i] > arr[i+1]) {
        var temp = arr[i];
        arr[i] = arr[i+1];
        arr[i+1] = temp;
        console.log(arr);
      }
    }
  }
  return arr;
}


// do not touch this
module.exports = bubbleSort;

